analytics/analytics/job_processor/__init__.py

```diff
@@ -121,7 +121,10 @@ def create_job_fact(
 
 @shared_task(
     name="process_job",
-    autoretry_for=(ReadTimeout,),
+    autoretry_for=(
+        ConnectionError,
```

Comment:
If we're already retrying the prometheus requests, what does adding an autoretry for `ConnectionError` here do?

Reply:
You're right, they both do the same thing. I thought the Prometheus DNS issue was a separate exception, but it's not; I think using the celery autoretry is simpler than building in retry logic with `requests`, so I backed the latter out.